prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Absence {
+  id                  Int      @id @default(autoincrement())
+  lessonDate          DateTime
+  subject             Subject
+  lessonPlan          String?
+  reasonOfAbsence     String
+  numberOfStudents    Int
+  absentTeacherId     Int
+  substituteTeacherId Int?
+  locationId          Int
+  absentTeacher       User     @relation("absentTeacher", fields: [absentTeacherId], references: [id])
+  substituteTeacher   User?    @relation("substituteTeacher", fields: [substituteTeacherId], references: [id])
+  location            Location @relation(fields: [locationId], references: [id])
+}
+
+model User {
+  id          Int       @id @default(autoincrement())
+  authId      String
+  email       String    @unique
+  firstName   String
+  lastName    String
+  role        Role      @default(TEACHER)
+  status      Status    @default(INVITED)
+  absences    Absence[] @relation("absentTeacher")
+  substitutes Absence[] @relation("substituteTeacher")
+}
+
+model Location {
+  id           Int    @id @default(autoincrement())
+  name         String
+  abbreviation String
+  absences     Absence[]
```

Comment:
I don't think we need to record absences per location at the moment. We can introduce this in the future if the client believes it to be helpful and we have enough time to implement it.
```suggestion
```

Reply:
its not actually recording absences per location, this is just necessary in order to have this line in the Absence schema.
`  location            Location @relation(fields: [locationId], references: [id])`.

We just need this to define the relationship beteween the two but `absences` isn't actually a column in the `Location` schema

Reply:
Thanks for clarifying!